pkg/deploy/reaper/reaper.go

```diff
@@ -55,9 +55,21 @@ func (reaper *DeploymentConfigReaper) Stop(namespace, name string, timeout time.
 		return err
 	}
 
-	// If there is neither a config nor any deployments, we can return NotFound.
+	options = kapi.ListOptions{LabelSelector: util.DeployerPodSelector(name)}
```

Comment:
[DeployerPodSelector](https://github.com/openshift/origin/blob/604ae179d78568dea1512e0b60fdf2b1d58b1ff1/pkg/deploy/util/util.go#L84) is modeled to work on an individual deployment. What you need here is [AnyDeployerPodSelector](https://github.com/openshift/origin/blob/604ae179d78568dea1512e0b60fdf2b1d58b1ff1/pkg/deploy/util/util.go#L91)